beast/plotting/plot_cmd_with_fits.py

```diff
@@ -60,8 +60,10 @@ def plot(
     # read in data
     data_hdu = fits.open(data_fits_file)
     data_table = data_hdu[1].data
+    data_hdu.close()
     beast_hdu = fits.open(beast_fits_file)
     beast_table = beast_hdu[1].data
+    beast_hdu.close()
```

Comment:
Thanks for catching that these hadn't been closed!  Would it be worth using `with fits.open() as...` instead?

Reply:
We could.  This works and is explicit.  Maybe better????  No real idea.